Samples/PackageWithExternalLocation/cppwinrt/PackageWithExternalLocationCppAppSetup/main.cpp

```diff
@@ -0,0 +1,80 @@
+﻿#include "pch.h"  
+#include <iostream>  
+#include <string>
+#include <winrt/Windows.Management.Deployment.h>  
+#include <winrt/Windows.ApplicationModel.Activation.h>  
+#include <shellapi.h>
+
+using namespace winrt;  
+using namespace winrt::Windows::Management::Deployment;  
+
+void InstallPackage(const std::wstring& packagePath, const std::wstring& externalLocation)
+{
+    try 
+    {
+        if (!externalLocation.empty() && !packagePath.empty())
+        {
+            winrt::Windows::Management::Deployment::PackageManager packageManager;
+            winrt::Windows::Management::Deployment::AddPackageOptions addOptions;
+            addOptions.ExternalLocationUri(winrt::Windows::Foundation::Uri(winrt::hstring(externalLocation)));
+
+            auto deploymentResult = packageManager.AddPackageByUriAsync(winrt::Windows::Foundation::Uri(winrt::hstring(packagePath)), addOptions).get();
+            auto extendedErrorCode = deploymentResult.ExtendedErrorCode();
+
+            if (extendedErrorCode != HRESULT_FROM_WIN32(ERROR_PACKAGES_IN_USE) && FAILED(extendedErrorCode))
+            {
+                if (extendedErrorCode == HRESULT_FROM_WIN32(CERT_E_UNTRUSTEDROOT))
+                {
+                    MessageBox(nullptr, L"Please install your certificate to Trusted Root Certification Authorities for Local Machine. Running without identity", L"result", MB_ICONINFORMATION);
+                }
+                else
+                {
+                    MessageBox(nullptr, L"Failed to register app with identity. Running without identity", L"result", MB_ICONINFORMATION);
+                }
+            }
+        }
+    }  
+    catch (const std::exception& ex) 
+    {
+        MessageBox(nullptr, L"Error during installation: ", L"ex.what()", MB_ICONERROR);
```

Comment:
```suggestion
        MessageBoxA(nullptr, "Error during installation: ", ex.what(), MB_ICONERROR);
```

(I switched to `MessageBoxA` because `ex.what` is not wide. You could try and convert it, but seems like too much for a sample...)

Reply:
Still quotes around `ex.what`